### PR TITLE
Commit maintenance message and PLP related updates

### PR DESCRIPTION
**Pull request by LianeHughes**

### CODE REVIEW

File: content/english/resources/bsl3.md [hunk 1]
```diff
@@ -1,33 +1,47 @@
 ---
 title: "BSL3 Biomedicum-SciLifeLab Collaborative Platform"
 category: "plp1"
-banner: "/resources_thumbs/bsl3_lab.jpg"
```
Comment (senthil10) [inline content/english/resources/bsl3.md:4]: I see the `bsl3_lab.jpg` is not removed, if it is not used elsewhere can we please remove it?

File: layouts/index.html [hunk 2]
```diff
@@ -12,6 +12,13 @@
       SciLifeLab Data Centre wish everyone a Merry Christmas and a Happy New Year! <i class="bi bi-snow2"></i> <i class="bi bi-snow2"></i><br> The team will be away from 20th December until 2nd January, and will respond to any questions/requests as soon as possible.
       {{ end }}
       </div> -->
+    <div style="color: white;" class="bg-success border rounded py-2 px-4 mb-3 fst-italic">
+      {{ if eq $.Site.Language.LanguageName "Svenska" }}
+      Underhåll av Pathogens Portal Sverige kommer att genomföras måndagen den 29 april.<br>Användare kan därför uppleva vissa problem med att komma åt webbplatsen.
+      {{ else }}
+      The Swedish Pathogens Portal will be undergoing maintenance on Monday 29th April.<br>Users may experience some issues accessing the site during this time. 
+      {{ end }}
```
Comment (senthil10) [inline layouts/index.html:20]: Is the migration gonna be done on April 29th ? or should we make the text more general ? For example something like, *"We are migrating to new cluster, so there is possibility of momentary unavailability of the site"*

File: static/resorces/bsl3-facility-crispr-based-antiviral.png
Comment (senthil10) [inline static/resorces/bsl3-facility-crispr-based-antiviral.png:1]: The file size is increased, is it ok if I compress it again ?

**Review by senthil10 — COMMENTED** [review]
Nice work, have some small comments/clarifications.

File: content/english/resources/bsl3.md (hunk 1 above)
Comment (LianeHughes) [reply content/english/resources/bsl3.md:4]: We can delete

**Review by senthil10 — COMMENTED** [review]
Once migration plan is finalised and the text is set accordingly, it can be merged

**Pull request merged**